include/ck/tensor_operation/gpu/device/device_reduce_multiblock.hpp

```diff
@@ -393,10 +394,8 @@ struct DeviceReduceMultiBlock : public DeviceReduce<InElementwiseOperation, AccE
         };
     };
 
-    bool IsSupportedArgument(const BaseArgument* p_arg) override
+    static bool IsSupportedArgument(const Argument* pArg)
```

Comment:
https://github.com/ROCmSoftwarePlatform/composable_kernel/blob/c7a96ed5e55e652fd2f0d1b2a4f52615b1a6fe87/include/ck/tensor_operation/gpu/device/device_gemm_reduce_xdl_cshuffle.hpp#L643-L649

Because of consistency. Also it's convenient for DeviceSoftmax to call DeviceReduce's checks without a reduction instance.